### PR TITLE
Update selenium.sh

### PR DESCRIPTION
**Pull request by vijayreddy1991**
https://github.com/Shippable/pm/issues/10588

### CODE REVIEW

File: version/selenium.sh [hunk 1]
```diff
@@ -27,14 +27,15 @@ sudo apt-get install -y \
   --no-install-recommends
 
 echo "================= Installing Chrome ==================="
-#sudo wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
-#sudo dpkg -i google-chrome-stable_current_amd64.deb
-#sudo apt-get install -y -f
-#sudo rm -f google-chrome-stable_current_amd64.deb
+sudo apt-get install libgconf2-4 libnss3-1d libxss1
```
Comment (ric03uec) [inline version/selenium.sh]: add -y here

Comment (vijayreddy1991) [reply version/selenium.sh]: ok

File: version/selenium.sh [hunk 2]
```diff
@@ -27,14 +27,15 @@ sudo apt-get install -y \
   --no-install-recommends
 
 echo "================= Installing Chrome ==================="
-#sudo wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
```
Comment (ric03uec) [inline version/selenium.sh]: remove commented code

Comment (vijayreddy1991) [reply version/selenium.sh]: ok

File: version/selenium.sh [hunk 3]
```diff
@@ -27,14 +27,11 @@ sudo apt-get install -y \
...
-#sudo dpkg -i google-chrome-stable_current_amd64.deb
-#sudo apt-get install -y -f
-#sudo rm -f google-chrome-stable_current_amd64.deb
-
-wget -q -O- https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
-sudo apt-get update
-sudo apt-get install google-chrome-stable
+sudo apt-get install -y libgconf2-4 libnss3-1d libxss1
```
Comment (ric03uec) [inline version/selenium.sh]: is this process working locally?

Comment (vijayreddy1991) [reply version/selenium.sh]: yes

**Pull request merged**